public/app/themes/justice/jsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
     "compilerOptions": {
         "jsx": "react-jsx"
-    }
+    },
+    "exclude": ["node_modules", "dist"]
 }
```

Comment:
Might need a new line at the end

Reply:
Cheers! This is sorted in the below commit.